src/services/celebration-service.ts

```diff
@@ -66,14 +66,18 @@ export class CelebrationService {
                             !blacklistData.includes(userData.UserDiscordId)
                     );
 
-                    let membersWithBirthdayTodayOrTomorrow = guildMembers.filter(member =>
-                        CelebrationUtils.isBirthdayTodayOrYesterday(
-                            memberUserDatas.find(data => data.UserDiscordId === member.id),
-                            guildData
-                        )
+                    let membersWithBirthdayTodayOrYesterday = guildMembers.filter(
+                        member =>
+                            !member.roles.cache
+                                .map(r => r.id)
```

Comment:
You should be able to do `member.roles.cache.keyArray()`

Reply:
keyArray() is no longer supported, I assume you mean `[...member.roles.cache.keys()]`?